src/resources/js/views/multiday-events.js

```diff
@@ -0,0 +1,179 @@
+	 *
+	 * @return {function} event handler for on hover out
+	 */
+	obj.onHoverOut = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.removeClass( obj.selectors.multidayEventInnerHover.className() );
+		};
+	};
+
+	/**
+	 * Add class to visible multiday event on hidden multiday event focus
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on focus
+	 */
+	obj.onFocus = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.addClass( obj.selectors.multidayEventInnerFocus.className() );
+		};
+	};
+
+	/**
+	 * Remove class to visible multiday event on hidden multiday event blur
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $visibleMultidayEventInner jQuery object of visible multiday event
+	 *
+	 * @return {function} event handler for on blur
+	 */
+	obj.onBlur = function( $visibleMultidayEventInner ) {
+		return function() {
+			$visibleMultidayEventInner.removeClass( obj.selectors.multidayEventInnerFocus.className() );
+		};
+	};
+
+	/**
+	 * Binds events for hover and focus of hidden multiday events
+	 *
+	 * @since TBD
+	 *
+	 * @param {number} index index of hidden multiday events from jQuery selector
+	 * @param {HTMLElement} hiddenMultidayEvent HTML element of hidden multiday event
+	 *
+	 * @return {void}
+	 */
+	obj.bindEvents = function() {
+		var $hiddenMultidayEvents = $( obj.selectors.multidayEvent );
+
+		$hiddenMultidayEvents.each( function( index, hiddenMultidayEvent ) {
+			var $hiddenMultidayEvent = $( hiddenMultidayEvent );
+			var $visibleMultidayEvents = obj.findVisibleMultidayEvents( $hiddenMultidayEvent );
+
+			$visibleMultidayEvents.each( function( index, visibleMultidayEvent ) {
+				var $visibleMultidayEvent = $( visibleMultidayEvent );
+				var $visibleMultidayEventInner = $visibleMultidayEvent.find( obj.selectors.multidayEventInner );
+				var $hiddenMultidayEventInner = $hiddenMultidayEvent.find( obj.selectors.multidayEventInner );
+
+				$hiddenMultidayEventInner.hover( obj.onHoverIn( $visibleMultidayEventInner ), obj.onHoverOut( $visibleMultidayEventInner ) );
+				$hiddenMultidayEventInner.focus( obj.onFocus( $visibleMultidayEventInner ) );
+				$hiddenMultidayEventInner.blur( obj.onBlur( $visibleMultidayEventInner ) );
+			} );
+		} );
+	};
+
+	/**
+	 * Handles the initialization of the multiday events when Document is ready
+	 *
+	 * @since TBD
+	 *
+	 * @return {void}
+	 */
+	obj.ready = function() {
+		/**
+		 * @todo: this isn't working... need to figure out why and fix.
+		 */
+		tribe.events.views.manager.$containers.on( 'afterSetup.tribeEvents', obj.bindEvents );
+		/**
+		 * @todo: do below for ajax events
+		 */
+		// on 'beforeAjaxBeforeSend.tribeEvents' event, remove all listeners
+		// on 'afterAjaxSuccess.tribeEvents' or 'afterAjaxError.tribeEvents', add all listeners
```

Comment:
With the container changes to `bindEvents` we wouldn't need to do those.

Reply:
probably need to keep `afterAjaxError`, right? if we're removing before send, then reinit on success or fail? since success runs `setup`, we don't need that one, but on fail, if we don't reinit, then functionality fails.